desktop/windows/wsl.md

```diff
@@ -49,19 +44,29 @@ Docker Desktop does not require any particular Linux distros to be installed. Th
 
     To check the WSL mode, run:
 
-     `wsl.exe -l -v`
+     ```consolse
```

Comment:
typo;

```suggestion
     ```console
```

Reply:
I think the indentation for these may be off as well; looks like there's spaces before the `$` prompt in the rendered output;

<img width="292" alt="Screenshot 2023-01-09 at 14 56 55" src="https://user-images.githubusercontent.com/1804568/211324817-3ea95d1c-2ed5-4509-a688-714debd9fcea.png">


That's probably because before it used "3 spaces" to mark it as a code-block, which is now replaced with the triple back-tick code-fences.

Reply:
It looks like this a CSS issue rather than the change I made. It appears the same in other places as well, like the Ubuntu page https://docs.docker.com/desktop/install/linux-install/

Think this fix is beyond the scope of this PR. Have fixed the typo though